rust/unicode/src/lib.rs

```diff
@@ -23,6 +23,10 @@ use core::cmp::Ordering;
 
 use crate::tables::*;
 
+/// The Unicode line breaking property of the given code point.
+///
+/// This is given as a numeric value which matches the ULineBreak
+/// enum value from ICU.
```

Comment:
Is it worth having an actual enum for this? could be `#[repr(u8)]`.

Reply:
I agree this would be better, but didn't take it on in this PR because it's a fairly major change, especially with all the autogeneration.

These are definitely expert level APIs. Also, as an evolution, it's likely that property value lookup will be moving to icu4x. So I'm not sure how much I want to invest in this long term.